execution/db-bootstrapper/src/bin/db_bootstrapper.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{ensure, format_err, Context, Result};
+use executor::db_bootstrapper::calculate_genesis;
+use libra_types::{transaction::Transaction, waypoint::Waypoint};
+use libra_vm::LibraVM;
+use libradb::LibraDB;
+use std::{fs::File, io::Read, path::PathBuf};
+use storage_interface::DbReaderWriter;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(
+    name = "db-bootstrapper",
+    about = "Calculate, verify and commit the genesis to local DB without a consensus among validators."
+)]
+struct Opt {
+    #[structopt(parse(from_os_str))]
+    db_dir: PathBuf,
+
+    #[structopt(short, long, parse(from_os_str))]
+    genesis_txn_file: PathBuf,
+
+    #[structopt(short, long)]
+    waypoint_to_verify: Option<Waypoint>,
+
+    #[structopt(long, requires("waypoint-to-verify"))]
+    commit: bool,
+}
+
+fn main() -> Result<()> {
+    let opt = Opt::from_args();
+
+    let genesis_txn = load_genesis_txn(&opt.genesis_txn_file)
+        .with_context(|| format_err!("Failed loading genesis txn."))?;
```

Comment:
Is it easier to just use `expect` instead of letting `main` return `Result`.

Reply:
The output is a touch cleaner :D . like
"
Error: Failed to calculate genesis.

Caused by:
Genesis txn didn't bump epoch.
"

vs 

"
thread 'main' panicked at 'Failed loading genesis txn.: No such file or directory (os error 2)', execution/db-bootstrapper/src/bin/db_bootstrapper.rs:35:23
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
"